docs/userGuide/tweakingThePageStructure.md

```diff
@@ -20,6 +20,7 @@
 
 <hr><!-- ======================================================================================================= -->
 
+<div id="layout">
```

Comment:
note this renders into

```
<div id="layout">
## Layouts // most markdown following a block html tag dosen't get markdown treatment
```

too, which is commonmark compliant

Reply:
Thanks for spotting this! Will be more careful from now on.